langkit/docs/modules.md

```diff
@@ -0,0 +1,273 @@
+# Modules List
+
+- [Injections](#injections)
+- [Input/Output](#inputoutput)
+- [Regexes](#regexes)
+- [Sentiment](#sentiment)
+- [Text Statistics](#text-statistics)
+- [Themes](#themes)
+- [Toxicity](#toxicity)
+
+
+## Injections
+
+The `injections` module gather metrics on possible prompt injection attacks. It will be applied to column named `prompt`, and it will create a new column named `prompt.injection`.
+
+### Usage
+
+```python
+from langkit import injections
```

Comment:
just a note, in other spots our examples show a pattern of `from langkit.<module> import *`, is there any difference in functionality and/or which should we update to be consistent?

Reply:
I discussed this with @jamie256 , and we should make this consistent by changing in the examples to use the pattern `from langkit import <module>` (will be done in a follow-up)